src/phantom.js

```diff
@@ -45,7 +36,36 @@ export default class Phantom {
         }
 
         let pathToShim = path.normalize(__dirname + '/shim.js');
-        logger.debug(`Starting ${phantomPath} ${args.concat([pathToShim]).join(' ')}`);
+
+        let loggerLevel = process.env.DEBUG === 'true' ? 'debug' : 'info';
+        if(typeof config.logger === 'object') {
```

Comment:
I tested it, it works.
However how to provide a default logger this way? Switching again to a global logger?

Also, if the default level is provided here, it would mean that the DEBUG environment variable would override its value, except if we remove it (which I think is a bad idea since it breaks the normal behavior), or if we put it outside of Phantom like logger was before and then use it here.